integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -228,6 +228,7 @@ <h3 class="heading font-bold text-black">
                             <i icon-name="tag" class="pb-1"></i>
                             {{ poi_form.category.label }}
                         </h3>
+                        ({% trans "currently not publicly visible" %})
```

Comment:
```suggestion
                        ({% trans "not visible in app" %})
```

Reply:
@ulliholtgrave are you sure you want to remove the "currently"? I can imagine that the users won't ever fill in these categories when they think that this is just for their own categorization and won't ever be shown in the app...
And it would be better if at least some of the categories would be filled in as soon as the app implements this feature, right? :thinking:

Reply:
Yeah, maybe you are right and we should keep the "currently". However, I would strip the "publicly" from the wording as it's also not available in some "private" area.

Reply:
okay, will do "currently not visible in the app"